tensorflow/tools/ci_build/ci_parameterized_build.sh

```diff
@@ -418,12 +418,12 @@ if [[ ${TF_BUILD_IS_PIP} == "no_pip" ]] ||
     BAZEL_TARGET=${TF_BUILD_BAZEL_TARGET}
   fi
 
-  if [[ ${CTYPE} == "cpu" ]] || \
+  if [[ ${CTYPE} == cpu* ]] || \
```

Comment:
I wasn't sure if I should wildcard cpu and gpu in these two files or if I should explicitly list cpu.ppc64le as an option like

```
 if [[ ${CTYPE} == "cpu" ]] || \
    [[ ${CTYPE} == "cpu.ppc64le" ]]; then \
    [[ ${CTYPE} == "debian.jessie.cpu" ]]; then
```

Reply:
To be honest, this file is in the works of being replaced. So either is OK with me.